multicluster/service-mirror/cluster_watcher.go

```diff
@@ -894,6 +894,10 @@ func (rcsw *RemoteClusterServiceWatcher) Start(ctx context.Context) error {
 					rcsw.log.Debugf("skipped processing endpoints object %s/%s: missing %s label", epNew.Namespace, epNew.Name, consts.DefaultExportedServiceSelector)
 					return
 				}
+				if rcsw.isRemoteDiscovery(epNew.Labels) {
```

Comment:
Do the endpoints of an exported service have the exported label on them?  Typically, the user would only label the exported service.  Does the endpoints controller copy this label to the endpoints or something like that?

Reply:
Yup, it appears labels are copied into the endpoints automatically:

```bash
$ k -n emojivoto expose pod web-8559b97f7c-pk6xq --port 80
service/web-8559b97f7c-pk6xq exposed

$ k -n emojivoto label svc/web-8559b97f7c-pk6xq mirror.linkerd.io/exported=remote-discovery

$ k -n emojivoto get ep web-8559b97f7c-pk6xq -ojson | jq .metadata.labels
{
  "app": "web-svc",
  "linkerd.io/control-plane-ns": "linkerd",
  "linkerd.io/proxy-deployment": "web",
  "linkerd.io/workload-ns": "emojivoto",
  "mirror.linkerd.io/exported": "remote-discovery",
  "pod-template-hash": "8559b97f7c",
  "version": "v11"
}
```